src/components/AddTodoForm.tsx

```diff
@@ -24,6 +24,7 @@ function AddTodoForm({ addTodo }: AddTodoFormProps) {
             <div>
                 <label>Title:</label>
                 <input
+                    value={title}
```

Comment:
In AddTodoForm.tsx, input required value={title} to show current state of the form that allow to control by the program. E.g. clear the value, or any manipulation. In this case, Form submission required to clear the input after add todo.